app/code/Magento/SampleData/Test/Unit/Model/DependencyTest.php

```diff
@@ -14,30 +16,36 @@
 use Magento\Framework\Filesystem;
 use Magento\Framework\Filesystem\Directory\ReadFactory;
 use Magento\Framework\Filesystem\Directory\ReadInterface;
-use Magento\Framework\Filesystem\DriverPool;
 use Magento\Framework\Phrase;
 use Magento\SampleData\Model\Dependency;
+use Magento\Framework\Filesystem\DriverPool;
 use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 
 /**
+ * Provides tests for Dependency model of SampleData module
+ *
+ * @covers \Magento\SampleData\Model\Dependency
```

Comment:
It's not crucial to provide that, but it's OK if you put effort on adding it :-)